content/contributor-guidelines.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Linebender contributor guidelines"
++++
+
+## About the projects
+
+The Linebender organization is an umbrella for a diverse and ambitious set of project, with the goal of creating high performance, accessible, high quality UI in Rust. An explicit goal is to push the boundary and fill in gaps in our understanding of UI infrastructure, as opposed to implementing existing cookbook techniques.
+
+## Discussions and community
+
+We use a [Zulip instance](https://xi.zulipchat.com/) as the primary place for discussions. All are welcome, and everyone with a GitHub account can sign up. Additional discussion happens in issues in the individual repos, and we also have tracking issues. We don’t use GitHub discussions, preferring the Zulip.
+
+We also hold weekly [office hours]. This is a chance to give status updates, make decisions, and discuss architectural directions. It is open to all, and there are public notes.
+
+We have recently re-started a [wiki], and eagerly encourage gardening. While the Zulip does have archives, they are not searchable, so the wiki represents the definitive place for the results of d
```

Comment:
I believe that the full history is searchable, although not indexed on external search engines